### PR TITLE
allow passing custom classes

### PR DESCRIPTION
**Pull request by LeZuse**
This allows you to pass custom classes to main and children buttons. Especially handy if you work with https://github.com/css-modules/css-modules

### CODE REVIEW

File: src/child-button.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
 'use strict';
 
 var React = require('react');
+var classnames = require('classnames');
 
 var ChildButton = React.createClass({
   render: function(){
-    var iconClass = 'mfb-component__child-icon ' + this.props.icon;
+    var iconClass = 'mfb-component__child-icon ' + this.props.icon,
```
Comment (phated) [inline src/child-button.js]: might as well use `classnames` here too.

File: src/child-button.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+        className = classnames('mfb-component__button--child', this.props.className);
```
Comment (phated) [inline src/child-button.js]: a `var` for each variable definition please

File: src/main-button.js [hunk 3]
```diff
@@ -13,18 +14,19 @@ var MainButton = React.createClass({
     };
   },
   render: function(){
-    var iconResting = 'mfb-component__main-icon--resting ' + this.props.iconResting,
-        iconActive = 'mfb-component__main-icon--active ' + this.props.iconActive;
+    var iconResting = classnames('mfb-component__main-icon--resting', this.props.iconResting),
+        iconActive = classnames('mfb-component__main-icon--active', this.props.iconActive),
```
Comment (phated) [inline src/main-button.js]: a `var` for each variable definition please

File: src/main-button.js [hunk 4]
```diff
@@ -13,18 +14,19 @@ var MainButton = React.createClass({
...
+        mainClass = classnames('mfb-component__button--main', this.props.className);
```
Comment (phated) [inline src/main-button.js]: a `var` for each variable definition please

**Comment by phated** [comment]
A few comments inline.  Awesome PR!

**Comment by LeZuse** [comment]
Totally agree with the proposed changes. Originally, I was just trying to go with the current code style.

**Comment by phated** [comment]
Published as 0.3.0.  Thanks!

**Pull request merged**